test/parallel/test-cluster-prof-and-logfile.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+const fs = require('fs');
+const assert = require('assert');
+const cluster = require('cluster');
+
+const waitForEvent = (eventEmiter, event) =>
+  new Promise((resolve) => eventEmiter.once(event, resolve));
+
+function findLogFileWithName(name, dir) {
+  const data = fs.readdirSync(dir, 'utf8');
+  return data.filter((file) => file.includes(name));
+}
+
+async function shouldFillLogfileArg() {
+  tmpdir.refresh();
```

Comment:
Do you need to call `tmpdir.refresh()` four times? Does it work if you just call it once at the beginning of the test? You don't need to clean up afterwards either.